src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -220,21 +220,6 @@ Copyright (c) .NET Foundation. All rights reserved.
     </RebuildDependsOn>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(MSBuildRuntimeType)' == 'Full'">
-    <!-- Automatically opt users into using the toolset package if they are running an MSBuild other than what this SDK was built against.
-          This is to reduce 'tearing'/dependency mismatch, but as always users can override this behavior by disabling the hosted compiler flag. -->
-    <BuildWithNetFrameworkHostedCompiler Condition="'$(BuildWithNetFrameworkHostedCompiler)' == ''
-                                                    and '$(_IsDisjointMSBuildVersion)' == 'true'
-                                                    and ('$(Language)' == 'C#'
-                                                    or '$(Language)' == 'VB')">true</BuildWithNetFrameworkHostedCompiler>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(BuildWithNetFrameworkHostedCompiler)' == 'true' and '$(OS)' == 'Windows_NT'">
-    <RoslynTargetsPath>$(NuGetPackageRoot)\microsoft.net.sdk.compilers.toolset\$(NETCoreSdkVersion)</RoslynTargetsPath>
-    <_NeedToDownloadMicrosoftNetSdkCompilersToolsetPackage>true</_NeedToDownloadMicrosoftNetSdkCompilersToolsetPackage>
-    <_MicrosoftNetSdkCompilersToolsetPackageRootEmpty Condition="'$(NuGetPackageRoot)' == ''">true</_MicrosoftNetSdkCompilersToolsetPackageRootEmpty>
-  </PropertyGroup>
-
   <!-- NOTE: Keep in sync with https://github.com/dotnet/msbuild/blob/main/src/Tasks/Microsoft.Common.tasks -->
   <!-- `Condition="Exists('$(RoslynTargetsPath)')` is needed because the package might not be yet downloaded during restore in VS
         and we do not want to fail the restore phase because of that. -->
```

Comment:
So the properties like `<RoslynTargetsPath>` were moved but the `<UsingTask>`s that actually use `$(RoslynTargetsPath)` and the rest of the logic remains here. How does that work?

Reply:
The common targets are imported before those here- https://github.com/dotnet/sdk/blob/23ab8b826eb8fc39a6a16635011779aed5a8d091/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L12

Reply:
I was more wondering how come we don't need the `<UsingTask>` and the other stuff in Common targets as well? Why only the properties are enough there?

Reply:
Yeah, the only thing used by APICompat in the outer build is the `RoslynTargetsPath`
https://github.com/dotnet/sdk/blob/23ab8b826eb8fc39a6a16635011779aed5a8d091/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ApiCompat.Common.targets#L29

Reply:
I see, thanks.